### PR TITLE
Mimecast 10082021

### PR DESCRIPTION
**Pull request by jason-miller0**
# PR Details

## Description

Added Mimecast folder
Added Mimecast.json
Added Mimecast.manifest.json
Added Screenshots
Added Readme.md

## Checklist

- [x] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ ] Updated CHANGELOG.md.
- [x] Ran unit tests locally.
- [x] Updated screenshots/sample logs if any.

### CODE REVIEW

**Comment by sumo-drew** [comment]
@jason-miller0 - Please fill out the "Documentation" section in the README.md file in the Mimecast folder according to the doc template here - https://docs.google.com/document/d/1Ae9nG--GXo0nkdCNMZIRZIuhCXwJyHvvAwe2Upc6kRo/edit.

For the "Collect Logs for Mimecast" section, you can just refer to our documentation as we have a built-in source to collect the logs, documented here - https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Cloud-to-Cloud_Integration_Framework/Mimecast_Source

File: Mimecast/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+# Mimecast
+
+Visualise Mimecast event log data ingested by Sumo Logic’s Hosted Collector and the Mimecast Source, and gain insight into activity occurring on your Mimecast tenant.
+
+# Documentation
```
Comment (himanshu219) [inline Mimecast/README.md:6]: Need to update readme

**Review by himanshu219 — CHANGES_REQUESTED** [review]
Apart from screenshots please add sample logs file and app icon.

**Comment by himanshu219** [comment]
@jason-miller0 the screenshots should be for full dashboard and can you please also provide some sample logs so we can test on our end

File: Mimecast/README.md (hunk 1 above)
Comment (jason-miller0) [reply Mimecast/README.md:117]: Done

**Comment by jason-miller0** [comment]
> @jason-miller0 - Please fill out the "Documentation" section in the README.md file in the Mimecast folder according to the doc template here - https://docs.google.com/document/d/1Ae9nG--GXo0nkdCNMZIRZIuhCXwJyHvvAwe2Upc6kRo/edit.
> 
> For the "Collect Logs for Mimecast" section, you can just refer to our documentation as we have a built-in source to collect the logs, documented here - https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Cloud-to-Cloud_Integration_Framework/Mimecast_Source
> 
> You can also check out the other partner folders as an example, thanks!



> Apart from screenshots please add sample logs file and app icon so that we can test the app.

Will add sample logs

**Comment by jason-miller0** [comment]
> @jason-miller0 the screenshots should be for full dashboard and can you please also provide some sample logs so we can test on our end

Full dashboard screenshots provided

**Comment by jason-miller0** [comment]
> > @jason-miller0 - Please fill out the "Documentation" section in the README.md file in the Mimecast folder according to the doc template here - https://docs.google.com/document/d/1Ae9nG--GXo0nkdCNMZIRZIuhCXwJyHvvAwe2Upc6kRo/edit.
> > For the "Collect Logs for Mimecast" section, you can just refer to our documentation as we have a built-in source to collect the logs, documented here - https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Cloud-to-Cloud_Integration_Framework/Mimecast_Source
> > You can also check out the other partner folders as an example, thanks!
> 
> > Apart from screenshots please add sample logs file and app icon so that we can test the app.
> 
> Will add sample logs

Sample logs have been added.

**Review by sumo-apps-team — APPROVED** [review]

**Pull request merged**